lib/cusparse/generic.jl

```diff
@@ -100,8 +100,10 @@ function mv!(transa::SparseChar, alpha::Number, A::CuSparseMatrixCSC{T}, X::Dens
     ctransa = 'N'
     if transa == 'N'
         ctransa = 'T'
+    elseif transa == 'C' && T <: Complex
+        error("Matrix-vector multiplication with the adjoint of a CSC matrix is not " *
```

Comment:
Maybe `ArgumentError` for this and the other `error` calls?

Reply:
Yes, makes sense. I have changed all `error` into `throw(ArgumentError(...))`.